B9PartSwitch/B9PartSwitch.csproj

```diff
@@ -202,4 +205,4 @@
   <Target Name="AfterBuild">
   </Target>
   -->
-</Project>
\ No newline at end of file
+</Project>
```

Comment:
Is this needed?  I prefer newlines at the ends of files too but it seems like Visual Studio removes it every time I save the project.

Reply:
That newline isn't an intentional change.  For whatever reason github did that to me automatically when I edited in the UnityEngine.TextRenderingModule reference.  I guess it's a quirk of the web interface.  Anyway, it should be fixed now.